src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -121,6 +121,11 @@ public class TraceLoadedDotNetRuntime
         /// Returns the .NET startup flags
         /// </summary>
         public StartupFlags StartupFlags { get; internal set; }
+        /// <summary>
+        /// Date and time of when the runtime was built
+        /// This is useful when a more detailed version is not present
+        /// </summary>
+        public DateTime RuntimeBuiltTime { get; internal set; }
```

Comment:
It turns out that null values are actually not as good as default values.   Typically user logic simply uses the value.  If it can be null, it throws an exception, and generally it is better just to use the 'obvious empty value'.    For just this reason we generally return empty strings rather than null strings.      You can argue about the merits of this, but for better or worse it is the convention of this library.